deploy/src/main/sandbox-k8s/letsencrypt-issuer-production.yml

```diff
@@ -0,0 +1,19 @@
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-prod
+spec:
+  acme:
+    # You must replace this email address with your own.
+    # Let's Encrypt will use this to contact you about expiring
+    # certificates, and issues related to your account.
+    email: TODO
```

Comment:
we might be able to use the hono-dev mailing list address. However, I will need to check with Eclipse webmasters if Let's Encrypt can actually send a mail to that address without having subscribed to the list.

Reply:
Any news here?

Reply:
The hono-dev mailing list does not accept mails from senders that are not subscribed to the list.

Reply:
OK. I changed the scripts and configuration files to make the mail address configurable so that we don't expose a personal email address.